java/src/Ice/src/main/java/com/zeroc/Ice/LocalException.java

```diff
@@ -4,13 +4,26 @@
 
 package com.zeroc.Ice;
 
-/** Base class for all Ice run-time exceptions. */
-public abstract class LocalException extends Exception {
+/** Base class for Ice run-time exceptions. */
+public class LocalException extends Exception {
   public LocalException() {}
```

Comment:
I would remove this parameterless constructor. The caller (including derived classes) must provide a message and/or a cause.

Reply:
Removed. Now all constructors require a `String message`.